docs/docs/dashboard.md

```diff
@@ -103,6 +129,18 @@ Folders can be searched by clicking on the search icon on the left drawer of the
 
 </div>
 
+### Slug for folders
+
+The slug for folders is automatically generated based on the name of the folder. The slug for folders can be used to access the folder directly from the URL. 
+
+Example: If the name of the folder is `customer support`, The folder can be accessed directly from the URL `https://tooljet.com/<workspace-name>?folder=customer%20support`.
```

Comment:
please add one more statement on how users can get this link.
- Basically user has to go to the folder to get the URL from the link in search bar of browser. Rephrase it better.